src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
-::The Gutter - Intro <1156,67>
-Anna was right. That piece of shit was trying to kill me one way or another. I guess I’d known it all along.
+<!-- If using debug menu shortcuts, prompts player to enter their name -->
+
+::The Gutter - Name prompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Outside approach]].{
+  (if: $playerName's last is "s")[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "c"))[
+    (set: $playerNamePlural to ($playerName + "es"))
+  ]
+  (else:)[
+    (set: $playerNamePlural to ($playerName + "s"))
+  ]
+}]
+}(else:)[No, [[that's not right|The Gutter - Name prompt]].]
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
+
+::The Gutter - Outside approach
+This was already too much shit for just one day. I kept my head down and my hood up, using what was left of my energy to keep moving. My boots trudged onward through the tarp-covered alleyways of Gatherville.
+
+(click: "boots trudged onward")[$dissolve[My gut lurched as I realized I still needed to ask around for more information on Anna. The last thing I wanted to do was talk to anyone but her.]]
+
+(click: "anyone but her")[$dissolve[I swallowed that feeling out of necessity, and fought to keep it and everything else down in my roiling gut. It was still pretty early in the night. Around this time, anyone who knew anything was:
+
+>>[[Getting fucked up at the Gutter|The Gutter - Outside intro]]
+>>[[Looking to score at the Roach Market|RM-first-entrance]]
+>>[[Catching an early show at the Church|Church - intro]]]]
+
+(set: $gutterOutside to "intro")(set: $spokeToToe to false)(set: $roachEntrance to "first")
+
+::The Gutter - Outside intro
+I stood outside a weathered, flat-roofed wooden building with a flickering neon sign.
+<!--This calls one of the three neon sign designs randomly every second. -->
+\
+(click: "neon sign")[(live: 1s)[(either: $guttersign1, $guttersign2, $guttersign3)]
```

Comment:
Minor thing, but in keeping with convention, these should be camel-case: `$gutterSign1`, etc.

Reply:
Fixed! I think I was so happy that I got this working that I totally missed this.